portal/src/components/Popup/CreateAccount/CreateAccount.component.js

```diff
@@ -0,0 +1,52 @@
+import Popup from '~/components/Popup'
+import { isNil } from 'lodash'
+export default {
+  name: 'create-account',
+  props: ['is-show', 'close', 'user'],
+  components: {
+    Popup
+  },
+  data() {
+    return {
+      is_submitting: false
+    }
+  },
+  methods: {
+    onCreateAccount() {
+      this.is_submitting = true
+      this.$store
+        .dispatch('postUser')
+        .then(() => {
+          setTimeout(() => {
```

Comment:
waarom is setTimeout nodig?

Reply:
je zou zeggen dat then en finally dat vanzelf wacht... ik kan het alleen niet testen, maar ik haal het weg